printer_test.go

```diff
@@ -402,6 +403,7 @@ fragment Foo on User {
 								{
 									Name: "hello",
 									Value: &ast.Value{
+										Raw:  "null",
```

Comment:
This is the only notable change to the tests. The rest is whitespace changes and named keys for the test cases.